pkg/runtime/lua.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runtime
+
+import (
+	"debug/elf"
+	"errors"
+	"fmt"
+)
+
+var luaJitIdentifyingSymbols = [][]byte{
+	[]byte("luaJIT"),
+}
+
+// HACK: This is a somewhat a brittle check.
+func IsLuaJit(f *elf.File) (bool, error) {
```

Comment:
It doesn't seem to be related to PR. Maybe a leftover?

Reply:
whops, yes, thanks